Automattic/jetpack
# Update/monetize jpcloud launchpad links

Related to https://github.com/Automattic/wp-calypso/pull/88778

> [!WARNING] 
> DO NOT MERGE until the related PR is merged :point_up_2: 
## Proposed changes:
As part of moving Monetize to Jetpack Cloud, Monetize launchpad tasks now link to Jetpack Cloud `/monetize` instead of WP `/earn`.

## Jetpack product discussion
Slack: p1711036278716469/1711029747.007349-slack-CKZHG0QCR

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:

* Choose an Atomic site.
* Install and activate [the Jetpack Beta plugin](https://jetpack.com/download-jetpack-beta/).
* Once the plugin is active, go to Jetpack > Jetpack Beta, select your plugin, and enable the `update/monetize-jpcloud-launchpad-links` branch on `WordPress.com Features`
* Remember to add `define( 'JETPACK_MU_WPCOM_LOAD_VIA_BETA_PLUGIN', true );` to your `wp-config.php` file.
* Follow this PR and check the launchpad links: https://github.com/Automattic/wp-calypso/pull/88778
* Launchpad links should link to Jetpack Cloud Monetize
